application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadAutoArchiver.java

```diff
@@ -129,8 +131,14 @@ private void handleArchiveFlow(ThreadChannel threadChannel, MessageEmbed embed)
             }
         };
 
+        logger.debug("Attempt to fetch thread-record from db, with ID: {} ", threadChannel.getId());
```

Comment:
was there a specific reason for extra unit of space? although it doesn't matter but to be consistent throughout.
```suggestion
        logger.debug("Attempt to fetch thread-record from db, with ID: {}", threadChannel.getId());
```

Reply:
No, consider it as a typo

Reply:
@ankitsmt211 i just noticed something thanks to what you said here, I think this _logger.debug()_ is unnecessary. As there is already a similar log inside  `helper.getThreadRecordByChannelId()`. So may be it's better to delete it 🤔

Reply:
Yep that seems reduntant, ideally it should reside in method handling db stuff only. I think you're good to remove it.